itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/AbstractCreateNewProjectTest.java

```diff
@@ -0,0 +1,132 @@
+package org.jboss.tools.quarkus.integration.tests.project;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.eclipse.ui.navigator.resources.ProjectExplorer;
+import org.eclipse.reddeer.eclipse.ui.problems.Problem;
+import org.eclipse.reddeer.eclipse.ui.views.markers.ProblemsView;
+import org.eclipse.reddeer.eclipse.ui.views.markers.ProblemsView.ProblemType;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.menu.ContextMenuItem;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.eclipse.reddeer.workbench.impl.editor.TextEditor;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.jboss.tools.quarkus.reddeer.common.QuarkusLabels.TextLabels;
+import org.jboss.tools.quarkus.reddeer.wizard.CodeProjectTypeWizardPage;
+import org.jboss.tools.quarkus.reddeer.wizard.QuarkusWizard;
+import org.junit.After;
+
+/**
+ * 
+ * @author olkornii@redhat.com
+ *
+ */
+public abstract class AbstractCreateNewProjectTest {
+
+	private static String jdkVersion11 = "11";
+	private static String propertyName = "java.specification.version";
+
+	private static String mavenCompileSourceStart = "<maven.compiler.source>";
+	private static String mavenCompileSourceEnd = "</maven.compiler.source>";
+	private static String mavenCompileTargetStart = "<maven.compiler.target>";
+	private static String mavenCompileTargetEnd = "</maven.compiler.target>";
```

Comment:
You have similar variables in file: CreateNewQuarkusConfigurationMavenTest.java...maybe we can create class AbstractQuarkusTest, where common variables will be defined? There could be also some before/after methods in the future(e.g. cleaning workspace....)

Reply:
fixed